Server.js

```diff
@@ -6,18 +6,21 @@ var winston = require('winston');
 var fs = require('fs');
 var tcpPortUsed = require('tcp-port-used');
 var http = require('http');
-var https = require('https');
 
-try {
-  fs.statSync('./config/config.js');
-} catch (error) {
-  console.log("Config file is not setup. Read README.md for config setup.");
-  return;
-}
-var config = require('./config/config');
-var models = require('./models');
-var log = require('./logging');
+var configPath = './config'
+process.argv.forEach((val, index) => {
+  if (val.toLowerCase().startsWith('config=')) {
+    configPath  = val.split('=')[1];
+  }
+});
```

Comment:
The convention for Unix-ish command line args is for multi-character options to start with `--` (i.e. `--config=`). This should really also fail if unsupported options are passed.

Instead of doing this all manually, try one of the many NodeJS packages available for command line processing. For simple command line option handling (like here) the following look promising:
* https://www.npmjs.com/package/minimist
* https://github.com/yargs/yargs
* https://github.com/sindresorhus/meow